mintpy/objects/stack.py

```diff
@@ -269,7 +269,7 @@ def read(self, datasetName=None, box=None, squeeze=True, print_msg=True):
                 box = [0, 0, self.width, self.length]
 
             # read
-            data = ds[dateFlag,
+            data = ds[np.ix_(dateFlag)[0],
```

Comment:
From my testing, the index issue is related to h5py v3+. I am using Python 3.8 with h5py-2.10, and it seems working fine for me. 

Assuming the h5py v3+ bug will be fixed in the future, I think it's better if we just wait and stick to version 2+ for now. #474 adds version constrain to h5py for the installation. Could you confirm if it works for you?

Reply:
I see, I have installed h5py separately and that is probably the issue. So shall I undo this one? or you will do it?

Reply:
If you could do it, please.